pkg/controller/certificates/sync.go

```diff
@@ -69,6 +69,8 @@ var (
 )
 
 func (c *Controller) Sync(ctx context.Context, crt *v1alpha1.Certificate) (err error) {
+	c.metrics.ControllerSyncCallCount.WithLabelValues(ControllerName).Inc()
```

Comment:
We typically wrap metric access in the `metrics` package in some more convenient functions. In this instance, there's not too much more to 'wrap' however for the sake of not exposing access to the actual metric counters (similar to other metrics), can you move the actual call to `Inc()` into the `metrics` package? It should make it easier for us to, for example, add a new label value to all references to this counter in future 😄